### DISCUSSION TITLE
Error while installing hmm: Command failed with status 1

### DISCUSSION THREAD

**Discussion by Zoe096423** [FunkinCrew/Funkin#3056] [Compiling help] [answered]
The following error happened during `hmm install`:
```
haxelib --never git lime https://github.com/FunkinCrew/lime 872ff6db2f2d27c0243d4ff76802121ded550dd7
Installing lime from https://github.com/FunkinCrew/lime branch: 872ff6db2f2d27c0243d4ff76802121ded550dd7
Error: Could not clone Git repository.
Execution error: command "haxelib --never git lime https://github.com/FunkinCrew/lime 872ff6db2f2d27c0243d4ff76802121ded550dd7" failed with status: 1
```

What happened for it to pop up?
Is there any possible way to solve this problem?

**Comment by NotHyper-474** [comment, marked as answer]
Never saw that before, the Lime repo is quite big so maybe a internet connection issue happened during the clone?

You could try running `hmm install` again

**Comment by amyspark-ng** [comment]
I'm getting this and i've re installed a few times, still not working

**Reply by NotHyper-474** [reply]
Can you try this?

![image](https://github.com/user-attachments/assets/cd3db4a9-3923-402c-88dd-1182150cf2e9)

**Reply by amyspark-ng** [reply]
![image](https://github.com/user-attachments/assets/21f8d338-6d35-4499-9daa-d2a4a78d8995)
got this for haxelib, hmm installed normally

**Reply by NotHyper-474** [reply]
Had the same error but seems like it doesn't impact anything. I think as long as `haxelib version` spits out `4.1.0 (2e623610985b580cbc49a389897619932e9d8586) (Funkin' Crew Edition)` it should be fine.
